citeproc/model.py

```diff
@@ -920,7 +918,6 @@ def process(self, date, context=None):
                 text = to_ordinal(date.day, context)
         elif name == 'month':
             form = self.get('form', 'long')
-            strip_periods = self.get('form', False)
```

Comment:
unused; I can add it back as a comment, or ... better ... put the variable to use.